website/docs/api/large-language-models.mdx

```diff
@@ -236,6 +236,57 @@ objects. This depends on the return type of the [model](#models).
 | `responses` | The generated prompts. ~~Iterable[Any]~~   |
 | **RETURNS** | The annotated documents. ~~Iterable[Doc]~~ |
 
+### Translation {id="translation"}
+
+The translation task translates texts from a defined or inferred source to a
+defined target language.
+
+#### spacy.Translation.v1 {id="translation-v1"}
+
+`spacy.Translation.v1` supports both zero-shot and few-shot prompting.
+
+> #### Example config
+>
+> ```ini
+> [components.llm.task]
+> @llm_tasks = "spacy.Translation.v1"
+> examples = null
+> target_lang = "Spanish"
+> ```
+
+| Argument                    | Description                                                                                                                                                                                   |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `template`                  | Custom prompt template to send to LLM model. Defaults to [summarization.v1.jinja](https://github.com/explosion/spacy-llm/blob/main/spacy_llm/tasks/templates/summarization.v1.jinja). ~~str~~ |
```

Comment:
TODO: update reference to jinja template

Reply:
Fixed in https://github.com/explosion/spaCy/commit/6265e0c52886525df21b39f8f762bfed91e7732d.